.gitignore

```diff
@@ -11,3 +11,4 @@ results.json
 lib/kubernetes_config_*
 tests/assets/eks-options.json
 tests/assets/do-options.json
+Dockerfile
```

Comment:
Not sure we need it?

Reply:
I am developing in a Docker container on Windows 10 that's why I added a Dockerfile to `.gitignore`